src/helpers/listOfPeopleWithParents.ts

```diff
@@ -0,0 +1,25 @@
+import { Person } from '../types';
+
+export const listOfPeopleWithParents = (people: Person[]): Person[] => {
+  // const copyListOfPeople = people.map(person => ({ ...person }));
```

Comment:
Don't leave unused code like this

Reply:
Thank you.
I forgot to push the correct commit.